container-images.txt

```diff
@@ -224,7 +224,7 @@ v1.30.0-upstream: docker.io/coredns/coredns:1.9.4 docker.io/kubernetesui/dashboa
-v1.31-static: docker.io/cdkbot/microbot-amd64:latest docker.io/cdkbot/microbot-arm64:latest docker.io/cdkbot/microbot-s390x:latest registry.k8s.io/defaultbackend-amd64:1.5 registry.k8s.io/defaultbackend-arm64:1.5 registry.k8s.io/defaultbackend-ppc64le:1.5 registry.k8s.io/defaultbackend-s390x:1.4 registry.k8s.io/pause:3.9 quay.io/external_storage/nfs-client-provisioner:v3.1.0-k8s1.11 registry.k8s.io/ingress-nginx/controller:v1.6.4 quay.io/kubernetes-ingress-controller/nginx-ingress-controller-ppc64le:0.20.0 docker.io/calico/node:v3.25.1 docker.io/calico/kube-controllers:v3.25.1 docker.io/calico/pod2daemon-flexvol:v3.25.1 docker.io/calico/cni:v3.25.1 gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.26.0 gcr.io/cloud-provider-vsphere/csi/release/driver:v3.0.0 gcr.io/cloud-provider-vsphere/csi/release/syncer:v3.0.0 registry.k8s.io/sig-storage/csi-attacher:v4.2.0 registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.7.0 registry.k8s.io/sig-storage/csi-provisioner:v3.4.0 registry.k8s.io/sig-storage/csi-resizer:v1.7.0 registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1 registry.k8s.io/sig-storage/livenessprobe:v2.9.0
+v1.31-static: docker.io/cdkbot/microbot-amd64:latest docker.io/cdkbot/microbot-arm64:latest docker.io/cdkbot/microbot-s390x:latest registry.k8s.io/defaultbackend-amd64:1.5 registry.k8s.io/defaultbackend-arm64:1.5 registry.k8s.io/defaultbackend-ppc64le:1.5 registry.k8s.io/defaultbackend-s390x:1.4 registry.k8s.io/pause:3.9 quay.io/external_storage/nfs-client-provisioner:v3.1.0-k8s1.11 registry.k8s.io/ingress-nginx/controller:v1.11.2 quay.io/kubernetes-ingress-controller/nginx-ingress-controller-ppc64le:0.20.0 docker.io/calico/node:v3.25.1 docker.io/calico/kube-controllers:v3.25.1 docker.io/calico/pod2daemon-flexvol:v3.25.1 docker.io/calico/cni:v3.25.1 gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.26.0 gcr.io/cloud-provider-vsphere/csi/release/driver:v3.0.0 gcr.io/cloud-provider-vsphere/csi/release/syncer:v3.0.0 registry.k8s.io/sig-storage/csi-attacher:v4.2.0 registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.7.0 registry.k8s.io/sig-storage/csi-provisioner:v3.4.0 registry.k8s.io/sig-storage/csi-resizer:v1.7.0 registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1 registry.k8s.io/sig-storage/livenessprobe:v2.9.0
```

Comment:
the diff on this line is subtle:

search for `registry.k8s.io/ingress-nginx/controller`